test/lib/alloc-cluster.js

```diff
@@ -155,6 +155,10 @@ function allocCluster(opts) {
         chan.listen(port, host);
         cluster.channels[i] = chan;
 
+
+        chan.setLazyRelaying(true);
```

Comment:
yeah, this can't land as is, instead you'll want to use our new paramterized testing feature, which we need to get a lazy and un-lazy mode into travis Soon ™.